docs-partials/builder/azure/arm/Config-not-required.mdx

```diff
@@ -234,6 +234,18 @@
   See [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/user-data)
   to learn more about user data.
 
+- `custom_script` (string) - Specify a command to inject into the CustomScriptExtension, to run on startup
+  on Windows builds, before the communicator attempts to connect
+  See [documentation](https://docs.microsoft.com/en-us/azure/virtual-machines/extensions/custom-script-windows)
+  to learn more.
+
+  The following example executes the contents of the file specified by `user_data_file`:
+
+  ```hcl2
+  custom_script   = "powershell -ExecutionPolicy Unrestricted -NoProfile -NonInteractive -Command \"$userData = (Invoke-RestMethod -Headers @{Metadata=$true} -Method GET -Uri http://169.254.169.254/metadata/instance/compute/userData?api-version=2021-01-01$([char]38)format=text); $contents = [System.Text.Encoding]::UTF8.GetString([Convert]::FromBase64String($userData)); set-content -path c:\\Windows\\Temp\\userdata.ps1 -value $contents; . c:\\Windows\\Temp\\userdata.ps1;\""
```

Comment:
Really appreciate the effort you've put into this example in the docs, this will really make this feature much more discoverable, great work @matt-richardson